### PR TITLE
[4.0.x] Updated source translation catalogs.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: django/contrib/admin/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -665,6 +665,14 @@ msgstr ""
 msgid "Toggle navigation"
 msgstr ""
 
+#: contrib/admin/templates/admin/nav_sidebar.html:5
+msgid "Start typing to filter..."
```
Comment (claudep) [inline django/contrib/admin/locale/en/LC_MESSAGES/django.po:669]: Should we change the original string to use `…` instead of 3 dots?

**Review by claudep — CHANGES_REQUESTED** [review]
Thanks for caring for this! I'm not always convinced it's worth updating files when only source code line numbers are updated. Apart from the patch size, this doesn't matter much.

**Comment by felixxm** [comment]
> I'm not always convinced it's worth updating files when only source code line numbers are updated.

Agreed, that's why I only included files with actual changes.

File: django/contrib/admin/locale/en/LC_MESSAGES/django.po (hunk 1 above)
Comment (felixxm) [reply django/contrib/admin/locale/en/LC_MESSAGES/django.po:669]: Good catch, #14875.

**Comment by felixxm** [comment]
@claudep Thanks for checking :+1: Updated.

**Review by claudep — APPROVED** [review]

**Pull request merged**